app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -213,36 +213,31 @@ def remove_subscriptions
 
     api :PUT, "/activation_keys/:id/content_override", N_("Override content for activation_key")
     param :id, :number, :desc => N_("ID of the activation key"), :required => true
-    param :content_override, Hash, :desc => N_("Content override parameters"), :deprecated => true do
-      param :content_label, String, :desc => N_("Label of the content"), :required => true
-      param :value, String, :desc => N_("Override to a boolean value or 'default'"), :required => true
-    end
-    param :content_overrides, Array, :desc => N_("Array of Content override parameters") do
+    param :content_overrides, Array, :desc => N_("Array of Content override parameters to be added in bulk") do
       param :content_label, String, :desc => N_("Label of the content"), :required => true
       param :value, String, :desc => N_("Override value. Provide a boolean value if name is 'enabled'"), :required => false
       param :name, String, :desc => N_("Override parameter key or name. Note if name is not provided the default name will be 'enabled'"), :required => false
       param :remove, :bool, :desc => N_("Set true to remove an override and reset it to 'default'"), :required => false
     end
     def content_override
-      content_overrides = []
-      if params[:content_override]
-        ::Foreman::Deprecation.api_deprecation_warning("The parameter content_override will be removed in Katello 3.5. Please update to use the content_overrides parameter.")
-        content_override_params = {:content_label => params[:content_override][:content_label]}
-        value = params[:content_override][:value]
-        if value == "default"
-          content_override_params[:remove] = true
-        elsif value
-          content_override_params[:value] = ::Foreman::Cast.to_bool(value)
+      if params[:content_overrides]
+        organization = @activation_key.organization
+        specified_labels = []
+        content_override_values = params[:content_overrides].map do |override|
+          specified_labels << override[:content_label]
+          validate_content_overrides_enabled(override)
+        end
+        specified_labels.uniq!
+        existing_labels = organization.contents.where(label: specified_labels)
+
+        unless specified_labels.size == existing_labels.size
+          missing_labels = specified_labels - existing_labels.pluck(:label).uniq
```

Comment:
I think content labels in an org are unique but I don't think there's harm in adding it. Though you might want to add the .uniq on the existing_labels relation so the size comparison is is .uniq on both sides.

Testing now!